foris/locale/de/LC_MESSAGES/foris.po

```diff
@@ -555,14 +557,14 @@ msgid ""
 "password. You can then set up your mobile devices, using the QR code "
 "available next to the form."
 msgstr ""
-"Wenn Sie den Router als Wi-Fi-Zugangspunkt verwenden wollen, dann aktivieren "
-"Sie die Wi-Fi-Option. Geben Sie dann die SSID (Netzwerkname) und ein "
+"Wenn Sie den Router als WLAN-Zugangspunkt verwenden wollen, dann aktivieren "
+"Sie die WLAN-Option. Geben Sie dann die SSID (Netzwerkname) und ein "
 "dazugehöriges Kennwort ein.<br><br>Danach können Sie Ihre Mobilgeräte "
 "mittels des angezeigten QR-Codes einfach konfigurieren."
 
 #, python-format
 msgid "Enable Wi-Fi %s"
-msgstr "Aktiviere Wi-Fi %s"
+msgstr "Aktiviere WLAN %s"
```

Comment:
Assuming the string "%s" is the SSID, we could also turn it into "%s aktivieren", leaving the term of Wi-Fi/WLAN out of this altogether, since it's in the Wi-Fi submenu. But I keep up my point, whenever we speak in Germany about wireless LAN, WLAN is the more prominent term. Wi-Fi is a term more present in the US.